tests/hmm/hmumu_utils.py

```diff
@@ -1440,6 +1445,8 @@ def get_selected_muons(
             mu_phi = NUMPY_LIB.array(mu_phi)
             mu_mass = NUMPY_LIB.array(mu_mass)
             mu_iso = NUMPY_LIB.array(mu_iso)
+
+        is_not_fsr = (muons.pt == mu_pt)
```

Comment:
This is for CPU part. maybe a line for the GPU part of the code can be added too?